modules/ROOT/pages/studio-update-sites.adoc

```diff
@@ -4,86 +4,34 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, esb, studio, anypoint studio, update sites, updates, extensions, plug-ins, plugins, download
 
-Update sites in Anypoint Studio are used to update or install Studio, the embedded Mule runtime, connectors or other useful development tools.
+Anypoint Studio uses update sites to update Studio, and the embedded versions of Mule runtime engine and MUnit.
```

Comment:
No comma; the comma could be interpreted to mean that Studio uses them to update itself...lol....
```suggestion
Anypoint Studio uses update sites to update both itself and embedded versions of Mule runtime engine and MUnit.
```